Resources/Prototypes/Entities/Clothing/OuterClothing/armor.yml

```
@@ -94,6 +94,8 @@
         Slash: 0.9
         Piercing: 0.9
         Heat: 0.4 # this technically means it protects against fires pretty well? -heat is just for lasers and stuff, not atmos temperature
+  - type: Reflect
+    reflectProb: 0.75
```

```suggestion
    reflectProb: 1
```

Assuming there is only one per map in the armory, and if they can't be vended, ordered, crafted, or printed, bump this up to a 100% chance. Was like that in ss13. Yes it's extremely powerful which is why there's only one per map, and generally why it's a traitor objective.

---

sure. i remember being able to order them with cargo egun crates on my old server so i went with 3/4

---

Just triple check there's only one per map and then you can make it 100%.